XRTK-Core/Packages/com.xrtk.core/Services/InputSystem/FocusProvider.cs

```diff
@@ -190,14 +190,6 @@ public PointerData(IMixedRealityPointer pointer)
 
             public void UpdateHit(PointerHitResult hitResult)
             {
-                if (hitResult.HitObject != CurrentPointerTarget)
-                {
-                    // Pointer.OnPreCurrentPointerTargetChange();
-
-                    // Set to default:
-                    Pointer.IsTargetPositionLockedOnFocusLock = true;
```

Comment:
So what sets this to true now, locking it to a specific target?

Reply:
In the inspectors or the developer toggles this flag in one of their scrips

Reply:
This didn't look like it was needed. I'm not entirely sure what it was there to begin with

Reply:
If we are unsure, then we should poll the community and even query it on Mrtk as to its use case and validity. As we do with any original code

Reply:
I think you missed the point.

This doesn't actually do anything afaik. I'm not interested in how the MRTK does it, or its usage there as it's likely been changed.